include/envoy/event/dispatcher.h

```diff
@@ -17,11 +17,29 @@
 #include "envoy/network/listen_socket.h"
 #include "envoy/network/listener.h"
 #include "envoy/network/transport_socket.h"
+#include "envoy/stats/scope.h"
+#include "envoy/stats/stats_macros.h"
 #include "envoy/thread/thread.h"
 
 namespace Envoy {
 namespace Event {
 
+/**
+ * All dispatcher stats. @see stats_macros.h
+ */
+// clang-format off
+#define ALL_DISPATCHER_STATS(HISTOGRAM)                                                            \
+  HISTOGRAM(loop_duration_us)                                                                      \
```

Comment:
I think we want microsecond resolution for these, but most other timing stats are measured in millis. Let me know if this is too weird.